### PR TITLE
The first version operator And tests the release workflow

### PR DESCRIPTION
**Pull request by 0x5457**

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package config
+
+import corev1 "k8s.io/api/core/v1"
+
+type Config struct {
+	PodMutator PodMutator `json:"podMutator"`
+}
+
+type PodMutator struct {
+	PatchStrategicMerge corev1.Pod      `json:"patchStrategicMerge"`
```
Comment (Code2Life) [inline internal/config/config.go:10]: For existing deployments already using GPU nodes, needs to remove its nvidia runtimeClass and gpu resource requests/limits,  this logic could also be configurable. default to remove nvidia gpu-operator native spec of gpu resource/runtimeClass part.

**Review by Code2Life — APPROVED** [review]
Awesome, LGTM for v0.1 !

File: api/v1/tensorfusionconnection_types.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Resource struct {
+	Tflops resource.Quantity `json:"tflops"`
```
[the remark this answers is not in the document]
Comment (0x5457) [reply api/v1/tensorfusionconnection_types.go:25]: This is planned for a future update.

File: internal/config/config.go (hunk 1 above)
Comment (0x5457) [reply internal/config/config.go:10]: This is planned for a future update.

**Pull request merged**